.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: 2
+
+jobs:
+  build:
+    working_directory: /tmp/code
+    docker:
+      - image: docker:18.02.0-ce
+    environment:
+      TZ: "/usr/share/zoneinfo/Etc/UTC"
+      TERM: dumb
+      DEBIAN_FRONTEND: "noninteractive"
+      # Definition of variables using other variables is not supported yet
+      # on CircleCi 2.0
+      # https://discuss.circleci.com/t/circle-2-0-global-environment-variables/8681/11
+      # For that reason we are storing the definition of variables
+      # in `variables` file - see `.circleci/variables`
+
+    steps:
+      # Checkout code
+      - checkout
+      # https://circleci.com/docs/2.0/building-docker-images/
+      - setup_remote_docker
+      - run:
+          name: Create a volume
+          command: |
+            docker create -v /src -v /mnt --name src alpine:3.7 /bin/true
+            docker cp . src:/src
+      - run:
+          name: Linting
+          command: |
+            docker run --rm -i --volumes-from src singapore/lint-condo /bin/sh -c \
```

Comment:
@netflash I am not familiar with lint-condo, so can you explain why the individual `docker run ..<linter>` calls are here, in addition to similar config in the lint-condo.yml file? My only guess is that the yaml file is not flexible enough to find files spread throughout the repo and thus the `find` calls are use to suss out matching files anywhere in the repo